src/components/Table/components/Row.tsx

```diff
@@ -376,4 +376,6 @@ export type GridDataRow<R extends Kinded> = {
   selectable?: false;
   /** Whether this row should infer its selected state based on its children's selected state */
   inferSelectedState?: false;
+  /** Fired when this row changes selected state. */
+  onSelect?: (isSelected: boolean) => void;
```

Comment:
Ah, fwiw @bdow I meant to include a 2nd opts arg that would pass the row, API, etc. Not sure if that makes a difference or not.

Reply:
Tbh, I don't see the need for a row level `onSelect`. I've always seen our `GridDataRow` objects being generated in a loop, where a row level `onSelect` seems excessive compared to the `onRowSelect` callback.
If we really _do_ need this row level callback, then I think it'd be worth adding the `row` as the 2nd param. Otherwise, it isn't super useful if the GridDataRows are generated within a loop. Though, again, I'd probably flag that as "maybe you should be using onRowSelect".